main/test/flix/Test.Handler.ZeroShot.Recursive.flix

```diff
@@ -0,0 +1,52 @@
+mod Test.Handler.ZeroShot.Recursive {
+
+    eff Throw {
+        pub def throw(): Unit
```

Comment:
Once https://github.com/flix/flix/pull/7047 is merged then `Unit` should become `Void`.